projects/simulation/human_dataset_generation/data_generator.py

```diff
@@ -0,0 +1,337 @@
+# Copyright 1996-2020 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ctypes
+import os
+import math
+import pyglet.gl
+from pywavefront import visualization, Wavefront
+import numpy as np
+from pyglet.window import pyglet
+import pickle
+import cv2
+import csv
+from background import Background
+
+
+class Data_generator(pyglet.window.Window):
```

Comment:
Just for consistency with the rest of the toolkit I think that it would be better if this class was renamed to `DataGenerator` instead of `Data_generator`.

Reply:
The class is now renamed, to all files.